gallery_dl/extractor/hentainexus.py

```diff
@@ -42,15 +42,42 @@ def metadata(self, page):
             "title"      : extr('<h1 class="title">', '</h1>'),
             "artist"     : rmve(extr('viewcolumn">Artist</td>'     , '</td>')),
             "book"       : rmve(extr('viewcolumn">Book</td>'       , '</td>')),
+            "circle"     : rmve(extr('viewcolumn">Circle</td>'     , '</td>')),
+            "event"      : rmve(extr('viewcolumn">Event</td>'      , '</td>')),
             "language"   : rmve(extr('viewcolumn">Language</td>'   , '</td>')),
             "magazine"   : rmve(extr('viewcolumn">Magazine</td>'   , '</td>')),
             "parody"     : rmve(extr('viewcolumn">Parody</td>'     , '</td>')),
             "publisher"  : rmve(extr('viewcolumn">Publisher</td>'  , '</td>')),
             "description": rmve(extr('viewcolumn">Description</td>', '</td>')),
         }
         data["lang"] = util.language_to_code(data["language"])
+        if 'doujin' in data["tags"]:
+            data["is_doujinshi"] = True
+        else:
+            data["is_doujinshi"] = False
+        data["conventional_title"] = self.join_title(
+            data["event"], data["circle"], data["artist"], data["title"], data["parody"], data["book"], data["magazine"]
+        )
         return data
 
+    @staticmethod
+    def join_title(event, circle, artist, title, parody, book, magazine):
+        jt = ''
+        if event:
+            jt += '({}) '.format(event)
+        if circle:
+            jt += '[{} ({})] '.format(circle, artist)
+        else:
+            jt += '[{}] '.format(artist)
+        jt += title
+        if parody.lower() != 'Original Work'.lower():
```

Comment:
Don't call `.lower()` on a string literal. Just do a comparison to the already lowercased string `original work`.